introduction.md

```diff
@@ -0,0 +1 @@
+This is dumb. Why does it have to be so long.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.